### PR TITLE
RavenDB-16974 Separate HTTP & TCP types drop-downs

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

File: src/Raven.Studio/typescript/viewmodels/manage/trafficWatch.ts [hunk 1]
```diff
@@ -132,21 +141,29 @@ class trafficWatch extends viewModelBase {
     
     attached() {
         super.attached();
-        awesomeMultiselect.build($("#visibleTypesSelector"), opts => {
+        
+        awesomeMultiselect.build($("#visibleTypesSelectorHttp"),this.getOptions(this.filteredTypeDataHttp));
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/manage/trafficWatch.ts]: space after `,`

File: src/Raven.Studio/wwwroot/Content/css/pages/traffic-watch.less [hunk 2]
```diff
@@ -18,4 +18,14 @@
     #trafficFilter {
         width: 220px;
     }
+    
+    .dropdown {
+        .dropdown-menu.stats-menu {
+            width: 850px;
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/Content/css/pages/traffic-watch.less]: can we avoid hardcoding width? Maybe use class=row, class=col-* instead?

File: src/Raven.Studio/typescript/viewmodels/manage/trafficWatch.ts (hunk 1 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/viewmodels/manage/trafficWatch.ts]: done

File: src/Raven.Studio/wwwroot/Content/css/pages/traffic-watch.less (hunk 2 above)
Comment (Danielle9897) [reply src/Raven.Studio/wwwroot/Content/css/pages/traffic-watch.less]: refactored

**Review by ml054 — APPROVED** [review]

File: src/Raven.Studio/wwwroot/App/views/manage/trafficWatch.html [hunk 3]
```diff
@@ -83,16 +82,29 @@
     <div class="panel margin-left margin-left-sm margin-right margin-right-sm">
         <div class="padding padding-lg margin-top margin-top-sm">
             <div class="row margin-top">
-                <div class="properties-container stats-container flex-horizontal padding padding-sm" data-label="Stats">
+                <div class="properties-container stats-container flex-horizontal padding padding-sm" data-label="General">
                     <div class="property">
-                        <h5><i class="icon-running-tasks"></i><span>Requests</span></h5>
-                        <strong data-bind="text: stats.count" class="stats-property"></strong>
+                        <h5><i class="icon-client"></i><span>Clients</span></h5>
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/App/views/manage/trafficWatch.html]: I wonder if label `Clients` is not ambiguous here. Let's suppose I have one application hosted on 3 nodes and it uses same certificate (app is load balanced) - then I'd say we have 1 client (I distinct certificate is used). 

Thoughts?

Comment (Danielle9897) [reply src/Raven.Studio/wwwroot/App/views/manage/trafficWatch.html]: '**Distinct IPs**' ? 
or
**Source IPs**  ?

=> Renamed to **`Source IPs`**

**Review by ml054 — APPROVED** [review]

**Pull request merged**